src/current/cockroachcloud/create-your-cluster.md

```diff
@@ -72,13 +69,39 @@ In the **Regions & nodes** section, select the number of nodes.
 
 Currently, you can add a maximum of 150 nodes to your cluster. For larger configurations, [contact us](https://support.cockroachlabs.com/hc/requests/new).
 
-## Step 5. Select the hardware per node
+Click **Next: Capacity**.
+
+## Step 4. Enable VPC Peering (optional)
```

Comment:
This content seems largely duplicated from https://www.cockroachlabs.com/docs/cockroachcloud/network-authorization. Is that intended?

Reply:
The content existed in this doc already and I'm only reorganizing it to match the new UX. Will look into the duplication in a follow-up PR if need be.